### DISCUSSION TITLE
How to handle click on message avatar

### DISCUSSION THREAD

**Discussion by alexey-zhulin** [flyerhq/flutter_chat_ui#159] [Ideas & Requests]
Could you suggest me how to handle click on message avatar?

**Comment by demchenkoalex** [comment]
You want to have a callback on avatar tap? I can add that

**Reply by alexey-zhulin** [reply]
Exactly 
Thank you

**Comment by demchenkoalex** [comment]
Added in [v1.5.2](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.5.2). (`onAvatarTap`)

**Reply by alexey-zhulin** [reply]
Thank you for the perfect solution
